### PR TITLE
Basic XBlock setup

### PR DESCRIPTION
**Pull request by mferenca**
Please review, @vkaracic & @marjev

### CODE REVIEW

File: google_maps/static/README.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Create a directory alongside this one named "public", and put files there.
+Then you can get a url with code like this:
+
+    url = self.runtime.local_resource_url(self, "public/js/lib.js")
+
+The sample code includes a function you can use to read the content of files
+in the static directory, like this:
+
+    frag.add_javascript(self.resource_string("static/js/my_block.js"))
+
```
Comment (vkaracic) [inline google_maps/static/README.txt]: Is this file necessary?

File: google_maps/translations/README.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Use this translations directory to provide internationalized strings for your XBlock project.
+
+For more information on how to enable translations, visit the Open edX XBlock tutorial on Internationalization:
+http://edx.readthedocs.org/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html
```
Comment (vkaracic) [inline google_maps/translations/README.txt]: Necessary?

File: google_maps/static/css/google_maps_edit.css [hunk 3]
```diff
@@ -0,0 +1 @@
+/* CSS for GoogleMapsXBlock studio view */
```
Comment (vkaracic) [inline google_maps/static/css/google_maps_edit.css]: I'm not much of a fan of adding empty files, makes more sense to me to add something once you have something to add. But I'm nitpicking here.

**Review by vkaracic — COMMENTED** [review]
A couple of questions. I'll have to refresh my knowledge of xblocks before commenting the code though.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+.. code:: bash
+
+	$ git clone https://github.com/ExtensionEngine/xblock-google-maps.git
+
+After you have cloned the repository you need to install the Google Maps 
+XBlock by running the following command:
+
+.. code:: bash
+
+	$ pip install -e xblock-google-maps
```
Comment (vkaracic) [inline README.rst:21]: Should we advise to install and activate a virtual environment first?

File: google_maps/static/README.txt (hunk 1 above)
Comment (mferenca) [reply google_maps/static/README.txt]: I guess not, deleted it.

File: google_maps/translations/README.txt (hunk 2 above)
Comment (mferenca) [reply google_maps/translations/README.txt]: I guess not, deleted it.

File: google_maps/static/css/google_maps_edit.css (hunk 3 above)
Comment (mferenca) [reply google_maps/static/css/google_maps_edit.css]: Definitely nitpicking, let's leave it like this for now.

File: README.rst (hunk 4 above)
Comment (mferenca) [reply README.rst:21]: I think not, this is not tutorial how to use pip.

Comment (vkaracic) [inline README.rst]: It's actually this xblock's installation guide.
@marjev virtualenv yay/nay?

**Review by vkaracic — APPROVED** [review]

**Comment by vkaracic** [comment]
Squash your commits.

**Review by marjev — APPROVED** [review]
